conans/client/printer.py

```diff
@@ -113,12 +113,6 @@ def _print(it_field, show_field=None, name=None, color=Color.BRIGHT_GREEN):
             _print("package_revision", name="Package revision", color=None)
             _print("binary", name="Binary", color=None)
 
-            if show("binary_remote") and is_ref:
```

Comment:
Why this removed but not the above ``if show("remote") and is_ref:``?

Reply:
Sorry, I missed this, removing